parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/transact.rs

```diff
@@ -0,0 +1,58 @@
+use crate::*;
+
+#[test]
+fn transact_sudo_from_relay_to_assets_para() {
+	// Init tests variables
+	// Call to be executed in Assets Parachain
+	const ASSET_ID: u32 = 1;
+
+	let call = <AssetHubWestend as Para>::RuntimeCall::Assets(pallet_assets::Call::<
```

Comment:
Would be nice to do this with foreign assets and register PAL.

Reply:
Absolutely. That kind of thing will be coming in future PRs.